src/language/utils.ts

```diff
@@ -14,111 +14,111 @@
+/**
+    * @returns true if any modifier kinds passed along exist in the given modifiers array
+    */
+export function hasModifier(modifiers: ts.ModifiersArray, ...modifierKinds: ts.SyntaxKind[]) {
+    if (modifiers == null || modifierKinds == null) {
+        return false;
     }
 
-    /**
-     * Determines if the appropriate bit in the parent (VariableDeclarationList) is set,
-     * which indicates this is a "let" or "const".
-     */
-    export function isBlockScopedVariable(node: ts.VariableDeclaration | ts.VariableStatement): boolean {
-        const parentNode = (node.kind === ts.SyntaxKind.VariableDeclaration)
-            ? (<ts.VariableDeclaration> node).parent
-            : (<ts.VariableStatement> node).declarationList;
+    return modifiers.some((m) => {
+        return modifierKinds.some((k) => m.kind === k);
+    });
+}
 
-        return isNodeFlagSet(parentNode, ts.NodeFlags.Let)
-            || isNodeFlagSet(parentNode, ts.NodeFlags.Const);
-    }
+/**
+    * Determines if the appropriate bit in the parent (VariableDeclarationList) is set,
+    * which indicates this is a "let" or "const".
+    */
+export function isBlockScopedVariable(node: ts.VariableDeclaration | ts.VariableStatement): boolean {
+    const parentNode = (node.kind === ts.SyntaxKind.VariableDeclaration)
+        ? (<ts.VariableDeclaration> node).parent
+        : (<ts.VariableStatement> node).declarationList;
 
-    export function isBlockScopedBindingElement(node: ts.BindingElement): boolean {
-        let currentParent = node.parent;
-        while (currentParent.kind !== ts.SyntaxKind.VariableDeclaration) {
-            if (currentParent.parent == null) {
-                // if we didn't encounter a VariableDeclaration, this must be a function parameter, which is block scoped
-                return true;
-            } else {
-                currentParent = currentParent.parent;
-            }
+    return isNodeFlagSet(parentNode, ts.NodeFlags.Let)
+        || isNodeFlagSet(parentNode, ts.NodeFlags.Const);
+}
+
+export function isBlockScopedBindingElement(node: ts.BindingElement): boolean {
+    let currentParent = node.parent;
+    while (currentParent.kind !== ts.SyntaxKind.VariableDeclaration) {
+        if (currentParent.parent == null) {
+            // if we didn't encounter a VariableDeclaration, this must be a function parameter, which is block scoped
+            return true;
+        } else {
+            currentParent = currentParent.parent;
         }
-        return isBlockScopedVariable(<ts.VariableDeclaration> currentParent);
     }
+    return isBlockScopedVariable(<ts.VariableDeclaration> currentParent);
+}
 
-    /**
-     * Bitwise check for node flags.
-     */
-    export function isNodeFlagSet(node: ts.Node, flagToCheck: ts.NodeFlags): boolean {
-        /* tslint:disable:no-bitwise */
-        return (node.flags & flagToCheck) !== 0;
-        /* tslint:enable:no-bitwise */
-    }
+/**
+    * Bitwise check for node flags.
+    */
```

Comment:
I looked into it - its output leaves a lot to be desired:
```cmd
 D:\rep\tslint ⮀ ⭠ modularize ⮀ grunt tsfmt
Running "tsfmt:src" (tsfmt) task
>> src/configuration.ts is not formatted
>> src/enableDisableRules.ts is not formatted
>> src/formatterLoader.ts is not formatted
>> src/formatters.ts is not formatted
>> src/formatters/index.ts is not formatted
>> src/formatters/jsonFormatter.ts is not formatted
>> src/formatters/pmdFormatter.ts is not formatted
>> src/formatters/proseFormatter.ts is not formatted
>> src/formatters/verboseFormatter.ts is not formatted
>> src/language/formatter/abstractFormatter.ts is not formatted
>> src/language/formatter/formatter.ts is not formatted
>> src/language/languageServiceHost.ts is not formatted
>> src/language/rule/abstractRule.ts is not formatted
>> src/language/rule/rule.ts is not formatted
>> src/language/utils.ts is not formatted
>> src/language/walker/blockScopeAwareRuleWalker.ts is not formatted
>> src/language/walker/index.ts is not formatted
>> src/language/walker/ruleWalker.ts is not formatted
>> src/language/walker/scopeAwareRuleWalker.ts is not formatted
>> src/language/walker/skippableTokenAwareRuleWalker.ts is not formatted
>> src/language/walker/syntaxWalker.ts is not formatted
>> src/lint.ts is not formatted
>> src/ruleLoader.ts is not formatted
>> src/rules.ts is not formatted
>> src/rules/alignRule.ts is not formatted
>> src/rules/banRule.ts is not formatted
>> src/rules/classNameRule.ts is not formatted
>> src/rules/commentFormatRule.ts is not formatted
>> src/rules/curlyRule.ts is not formatted
>> src/rules/eoflineRule.ts is not formatted
>> src/rules/forinRule.ts is not formatted
>> src/rules/indentRule.ts is not formatted
>> src/rules/interfaceNameRule.ts is not formatted
>> src/rules/jsdocFormatRule.ts is not formatted
>> src/rules/labelPositionRule.ts is not formatted
>> src/rules/labelUndefinedRule.ts is not formatted
>> src/rules/maxLineLengthRule.ts is not formatted
>> src/rules/memberAccessRule.ts is not formatted
>> src/rules/memberOrderingRule.ts is not formatted
>> src/rules/noAnyRule.ts is not formatted
>> src/rules/noArgRule.ts is not formatted
>> src/rules/noBitwiseRule.ts is not formatted
>> src/rules/noConditionalAssignmentRule.ts is not formatted
>> src/rules/noConsecutiveBlankLinesRule.ts is not formatted
>> src/rules/noConsoleRule.ts is not formatted
>> src/rules/noConstructRule.ts is not formatted
>> src/rules/noConstructorVarsRule.ts is not formatted
>> src/rules/noDebuggerRule.ts is not formatted
>> src/rules/noDuplicateKeyRule.ts is not formatted
>> src/rules/noDuplicateVariableRule.ts is not formatted
>> src/rules/noEmptyRule.ts is not formatted
>> src/rules/noEvalRule.ts is not formatted
>> src/rules/noInferrableTypesRule.ts is not formatted
>> src/rules/noInternalModuleRule.ts is not formatted
>> src/rules/noRequireImportsRule.ts is not formatted
>> src/rules/noShadowedVariableRule.ts is not formatted
>> src/rules/noStringLiteralRule.ts is not formatted
>> src/rules/noSwitchCaseFallThroughRule.ts is not formatted
>> src/rules/noTrailingCommaRule.ts is not formatted
>> src/rules/noTrailingWhitespaceRule.ts is not formatted
>> src/rules/noUnreachableRule.ts is not formatted
>> src/rules/noUnusedExpressionRule.ts is not formatted
>> src/rules/noUnusedVariableRule.ts is not formatted
>> src/rules/noUseBeforeDeclareRule.ts is not formatted
>> src/rules/noVarKeywordRule.ts is not formatted
>> src/rules/noVarRequiresRule.ts is not formatted
>> src/rules/oneLineRule.ts is not formatted
>> src/rules/quotemarkRule.ts is not formatted
>> src/rules/radixRule.ts is not formatted
>> src/rules/semicolonRule.ts is not formatted
>> src/rules/sortObjectLiteralKeysRule.ts is not formatted
>> src/rules/switchDefaultRule.ts is not formatted
>> src/rules/tripleEqualsRule.ts is not formatted
>> src/rules/typedefRule.ts is not formatted
>> src/rules/typedefWhitespaceRule.ts is not formatted
>> src/rules/useStrictRule.ts is not formatted
>> src/rules/variableNameRule.ts is not formatted
>> src/rules/whitespaceRule.ts is not formatted
>> src/tslint-cli.ts is not formatted
>> src/tslint.ts is not formatted
Warning: Task "tsfmt:src" failed. Use --force to continue.

Aborted due to warnings.
```

At least until it can suggest how to _fix_ those formatting errors (beyond allowing it to autoformat the code) I'm not sure it's as fit for purpose as just tslint itself is.

Given that it's just an additional proofing step, I don't think it needs to be in this PR, regardless.